routes/web.php

```diff
@@ -129,11 +123,14 @@
     Route::delete('delete-Job/{id}', [CompanyController::class, 'destroyJob'])
         ->name('all-jobs.destroy');
 
```

Comment:
Add a new route to show all the applications related to one job
1. URL `jobs/{job_id}/applications`
2. name `company.jobs.applications`